src/Model/Actions.re

```diff
@@ -80,6 +84,7 @@ type t =
   | EditorScrollToColumn(EditorId.t, int)
   | ShowNotification(Notification.t)
   | HideNotification(Notification.t)
+  | ClearNotifications
```

Comment:
This doesn't seem like an 'Event' either - this seems to describe the _intent_, which is really just a different naming scheme for a function. I believe what we'd want is to describe the event that caused it, vs the intent.

It could be I'm missing nuance, or that this is just set up to fit into the existing notification actions `ShowNotification` and `HideNotification` (which also aren't very 'Event-like'). It might be that we could tackle those events holistically if we move notifications to a feature. 

Just want to double-check my understanding of the 'Event' model we are using-

Reply:
Yeah, this is just to be consistent with what was already there. Both `Notifications` and `StatusBar` should be refactored into feature projects I think, but I'm not entirely sure where the boundary between them will be yet. Which project the notification count and context menu should live in, which will impact all these actions. I think they might just become function though, to be triggered by events in other projects.

Reply:
Cool, that makes sense! As we start to untangle the obvious parts into features, maybe some of these will fall out.